phpstan.neon.dist

```diff
@@ -1,14 +1,14 @@
 parameters:
     level: 8
-    phpVersion: 80200
+    phpVersion: 80300
     paths:
         - src
         - static-analysis
     treatPhpDocTypesAsCertain: false
     reportUnmatchedIgnoredErrors: false
-    checkMissingIterableValueType: false
-    checkGenericClassInNonGenericObjectType: false
     ignoreErrors:
+        - identifier: missingType.generics
```

Comment:
This is the replacement for `checkGenericClassInNonGenericObjectType: false`. There's too many errors when I just remove the setting, so I decided to keep ignoring this kind of error. We might want to revisit this issue on a higher branch though.